plugins/experimental/wasm/wasm_main.cc

```diff
@@ -62,23 +66,23 @@ transform_handler(TSCont contp, ats_wasm::TransformInfo *ti)
 
   ats_wasm::Context *c;
 
-  Dbg(dbg_ctl, "[%s] transform handler begins", __FUNCTION__);
+  Dbg(ats_wasm::dbg_ctl, "[%s] transform handler begins", __FUNCTION__);
```

Comment:
would it be more consistent (with the other plugins/definitions) to add `using namespace ats_wasm` instead of explicitly defining the namespace for each debug statement?

Reply:
I probably can. But I was just trying to make it consistent inside this file (wasm_main.cc) and not using namespace.